Monika After Story/game/script-topics.rpy

```diff
@@ -17567,6 +17567,54 @@ label monika_ddlcroleplay:
     m 1kub "Just make sure to remember that those versions of me are just stories, ahaha~"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_root_privileges",
+            conditional=(
+                "mas_hasRootPrivileges()"
+            ),
+            action=EV_ACT_PUSH,
+        )
+    )
+
+# Ever had root access/admin rights?
+default persistent._mas_pm_had_root_privileges = False
+
+label monika_root_privileges:
+    $ persistent._mas_pm_had_root_privileges = True
+
+    m 2lsc "Hey, [player]..."
+    m 2eud "It's hard to explain, but I... {w=0.5}somehow, I feel like I can do a lot more with your system now that I {i}couldn't do{/i} before."
+
+    show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve_monika
+    m 5rtc "Hmm, could it be... What was it?"
+    m 5dsc "..."
+
+    if renpy.windows:
+        m 5wud "{i}The admin rights?..{/i}"
+    else:
+        m 5wud "{i}The superuser rights?..{/i}"
+
+    if mas_isMoniNormal(higher=True):
+        show monika 1lkbsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+        m 1lkbsa "[player], I'm really happy that you trust me so much... {w=0.5}{nw}"
+        extend 3hubsb "but I don't even know what I could do with such privileges!"
```

Comment:
Not sure she doesn't know what she _could_ do, it's mainly what she would do with them
```suggestion
        extend 3hkbsb "but I don't even know what I would do with these privileges!"
```